api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3218,6 +3218,21 @@ enum OSType {
   OS_TYPE_WINDOWS = 3;
 }
 
+// Origin of a device.
```

Comment:
```suggestion
// DeviceOrigin is the same as teleport.devicetrust.v1.DeviceOrigin.
// Duplicated because gogo doesn't play well with protoc-gen-go.
```